src/main/java/com/minecolonies/commands/DeleteColonyCommand.java

```diff
@@ -0,0 +1,129 @@
+package com.minecolonies.commands;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.colony.IColony;
+import com.mojang.authlib.GameProfile;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * List all colonies.
+ */
+public class DeleteColonyCommand extends AbstractSingleCommand
+{
+
+    public static final  String DESC                       = "delete";
+    private static final String ID_TEXT                    = "§2ID: §f";
+    private static final String NAME_TEXT                  = "§2 Name: §f";
+    private static final String MAYOR_TEXT                 = "§2Mayor: §f";
+    private static final String COORDINATES_TEXT           = "§2Coordinates: §f";
+    private static final String COORDINATES_XYZ            = "§4x=§f%s §4y=§f%s §4z=§f%s";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "COORDINATES_XYZ" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)